react/features/prejoin/components/dialogs/DialInDialog.js

```diff
@@ -0,0 +1,121 @@
+// @flow
+
+import React from 'react';
+import { translate } from '../../../base/i18n';
+import { Icon, IconArrowLeft } from '../../../base/icons';
+import { getCountryCodeFromPhone } from '../../utils';
+import ActionButton from '../buttons/ActionButton';
+import Label from '../Label';
+
+type Props = {
+
+    /**
+     * The number to call in order to join the conference.
+     */
+    number: string,
+
+    /**
+     * Handler used when clicking the back button.
+     */
+    onBack: Function,
+
+    /**
+     * Click handler for the text button.
+     */
+    onTextButtonClick: Function,
+
+    /**
+     * Click handler for primary button.
+     */
+    onPrimaryButtonClick: Function,
+
+    /**
+     * Click handler for the small additional text.
+     */
+    onSmallTextClick: Function,
+
+    /**
+     * The pin of the conference.
+     */
+    pin: string,
```

Comment:
pin, password, passcode, ... let'ss pick one and be consistent? I think "passcode" is the one now

Reply:
Changed it. Thanks!